src/CachedCalls.jl

```diff
@@ -1,5 +1,86 @@
 module CachedCalls
 
-# Write your package code here.
+using FilePathsBase
+using FilePathsBase: /
+using JLSO
+
+const CACHEDCALLS_DIR = home() / ".cachedcalls"
```

Comment:
This needs to change to allow the user to specify their own directory.

One idea is to use environment variables which can be specified in the script that the user runs. Is that a good way to go about it? On downside is that the ENV is stored as a string which makes directory manipulation somewhat more fiddly.

Reply:
You could do something like this, where you set a default and give the user an option to override?
Also combine that with maybe an env var for it to persist?

> https://github.com/JuliaCloud/AWS.jl/blob/3dee0519b568c49232ae0b753ca3d5a5fb01d484/src/AWS.jl#L31-L67

Reply:
maybe put it under `.julia/` like DataDeps does? 
https://github.com/oxinabox/DataDeps.jl/blob/98e2f708a7fb798fce67269d2b7a791441370fd9/src/locations.jl#L4-L19

Reply:
Thanks both, this was exactly what I was looking for!